build.sbt

```diff
@@ -17,6 +17,7 @@ scalacOptions := Seq(
   "-language:_",
   "-Xlog-reflective-calls")
 
+// SBT requires 2.10 for now (1/15/15).
```

Comment:
So, this sets the default version within the core dependencies plugin, but not for the plugins library itself? That makes sense.